src/ResourceManager/Sql/Commands.Sql/Auditing/Cmdlet/GetAzureSqlServerAuditingPolicy.cs

```diff
@@ -22,6 +23,7 @@ namespace Microsoft.Azure.Commands.Sql.Auditing.Cmdlet
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "AzureRmSqlServerAuditingPolicy", SupportsShouldProcess = true), OutputType(typeof (AuditingPolicyModel))]
     [Alias("Get-AzureRmSqlDatabaseServerAuditingPolicy")]
+    [Obsolete]
```

Comment:
same comment:  Needs name of cmdlet, that iit is obsolete, and poiints the user at the replacement cmdlet